app/controllers/key_pairs_controller.rb

```diff
@@ -60,11 +92,15 @@ def update
 
     respond_to do |format|
       if @key_pair.update_attributes(key_pair_params)
-        format.html { redirect_to @key_pair, notice: 'Key pair was successfully updated.' }
+        format.html do redirect_to @key_pair,
+                                   notice: 'Key pair was successfully updated.'
+                    end
         format.json { head :no_content }
       else
         format.html { render action: "edit" }
-        format.json { render json: @key_pair.errors, status: :unprocessable_entity }
+        format.json do render json: @key_pair.errors,
+                             status: :unprocessable_entity
+                    end
```

Comment:
`end` at 103, 20 is not aligned with `format.json do render json: @key_pair.errors,` at 101, 8